expo/scheduler/views.py

```diff
@@ -911,6 +914,9 @@ def edit_event_display(request, item, errorcontext=None):
         conference=item.eventitem.get_conference(),
         date=item.starttime.date())
     initial['time'] = item.starttime.strftime("%H:%M:%S")
+    #initial['time'] = item.starttime.strftime("%T")
+    #initial['time'] = date_format(item.starttime, "TIME_FORMAT")
+    #initial['time'] = item.starttime
```

Comment:
Was keeping them until I got everything working, so I could return them easily enough.  No need to keep them now.